src/test/java/com/elastic/support/diagnostics/commands/TestCheckKibanaVersion.java

```diff
@@ -0,0 +1,151 @@
+package com.elastic.support.diagnostics;
+
+import com.elastic.support.Constants;
+import com.elastic.support.util.*;
+import org.junit.jupiter.api.Test;
+import com.elastic.support.diagnostics.commands.CheckKibanaVersion;
+import com.elastic.support.diagnostics.chain.DiagnosticContext;
+import org.mockserver.integration.ClientAndServer;
+import org.mockserver.matchers.Times;
+import org.mockserver.model.Header;
+import org.mockserver.model.Parameter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.*;
+import org.apache.http.client.methods.*;
+import org.apache.http.impl.client.*;
+import com.elastic.support.rest.RestClient;
+import com.elastic.support.rest.RestEntry;
+import com.vdurmont.semver4j.Semver;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockserver.integration.ClientAndServer.startClientAndServer;
+import static org.mockserver.model.HttpRequest.request;
+import static org.mockserver.model.HttpResponse.response;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class TestCheckKibanaVersion {
+
+	private static final Logger logger = LogManager.getLogger(RestClient.class);
```

Comment:
Can we reformat this class for removing broken indention? Only this file has several places with different indention.